conda-store-server/conda_store_server/orm.py

```diff
@@ -513,10 +513,11 @@ def update_packages(self, db, subdirs=None):
                 package_builds[package_key].append(new_package_build_dict)
             logger.info("CondaPackageBuild objects created")
 
-            batch_size = 1000
+            # sqlite3 has a max expression depth of 1000
+            batch_size = 990
```

Comment:
I bet there is an interesting story here? Is this only something that showed up on Windows?

Reply:
I haven't seen this personally, but it's mentioned in one of the Aaron's commits:

```
Lower the batch size for updating packages from 1000 to 990
1000 hits the sqlite3 max expression depth and results in an error.
```

Seems harmless, so I decided to keep it.